libs/core/src/lib/assets/fonts/sap_fiori_3_fonts.css

```diff
@@ -0,0 +1,6 @@
+@font-face {
```

Comment:
I think more correct way would be to move all css-related code to fundamental-styles theming. What do you think?

Reply:
If you mean fonts - agree, we could do this.

But common css `libs/core/src/lib/assets/fundamental-ngx-core.scss` probably should be kept in ngx to have ability to add any code that might be not present in styles.

Reply:
https://github.com/SAP/fundamental-styles/pull/3764